packages/manager/apps/zimbra/public/translations/common/Messages_fr_FR.json

```diff
@@ -54,7 +54,16 @@
   "order_zimbra_accounts": "Commander des comptes Zimbra",
   "form_mandatory_fields": "Les champs mentionnés avec un astérisque * sont obligatoires.",
   "form_max_chars": "{{ value }} caractères maximum",
+  "form_min_chars": "{{ value }} caractères minimum",
```

Comment:
```suggestion
```
   Can't we use "max_chars": "{{ value }} caractères maximum" 
  "min_chars": "{{ value }} caractères minimum" from common-translations ?

Reply:
it's planned to use these soon